docs/pipeline-standards.md

```diff
@@ -15,241 +15,244 @@ Libuit KG, Guthrie J, Ambrosio F, Kapsak C, Unal Gultekin, Holmes J, Wright S, N
+#### 10a. Contribution, Authorship, and Verified Point of Contact
+_Does the full list of authors seem appropriate and include a verified point of contact?_
 
-The description of functionality and performance testing should be made accessible via the code repository where the software is made available.
+Clearly listing authorship and credit in public health bioinformatics acknowledges the contributions of individual researchers, fostering a sense of ownership and responsibility for their work. This practice also promotes transparency, collaboration, and recognition within the scientific community, enhancing career development opportunities and encouraging the sharing of expertise. We would recommend using the [CRedIT system](https://credit.niso.org/) adopted by the Natural Sciences field to acknowledge contributions to bioinformatic tools. Contributors to the pipeline must be acknowledged as a co-author if they have contributed by: programming, pipeline development; designing computer programs; implementation of the computer code and supporting algorithms; testing of existing code components.
 
-- To meet this standard, provide a description of both automated and/or manual tests that assess the functionality of the software, ensuring reliability and accuracy.
-- To verify this standard, the reviewer should check the documentation for a description of tests, both automated and manual, that evaluate the functionality of the software, contributing to user confidence and consistent research outcomes.
+A verified point of contact must include a working email address of an individual or organization that is most likely to maintain the bioinformatic tool in the long term. Ideally, email addresses for multiple individuals should be provided and these should not be organizational email addresses (e.g. joe.bloggs@phag4e.org), as they could lose access to that email when they leave that organization. 
 
+**To adhere to this documentation practice:** Clearly list authorship and credit for the bioinformatics tool, acknowledging individual contributions and following the CRedIT system for appropriate recognition based on specific contributions.
 
-### Community Guidelines for Contribution and Support
-_Are there clear guidelines for third parties wishing to 1) contribute to the software 2) report issues or problems with the software and 3) seek support?_
+**To verify adherence to this documentation practice:** The reviewer should check the documentation and repository for a clear and comprehensive list of authors, ensuring adherence to the CRedIT system and appropriate acknowledgment of contributors based on their roles in programming, pipeline development, design, implementation, and testing.
```

Comment:
If to verify this we are going to say `ensuring adherence to the CRedIT system`, maybe in the text we should change `We would recommend using the [CRedIT system]`. It is not a recommendation if you require it.